.changeset/rich-insects-scream.md

```diff
@@ -0,0 +1,44 @@
+---
+'@astrojs/sitemap': minor
+---
+
+Add configurable XML namespaces support to sitemap generation
+
+- Add `namespaces` option to control which XML namespaces are included
+- Support for news, xhtml, image, and video namespaces
+- All namespaces enabled by default for backward compatibility
+
+Example: Removing the video namespace
+If you want to exclude the video namespace from your sitemap, set video: false in your configuration:
+
+```
+// astro.config.mjs
+import { sitemap } from '@astrojs/sitemap';
+
+export default {
+  integrations: [
+    sitemap({
+      namespaces: {
+        video: false,
+        // other namespaces remain enabled by default
+      }
+    })
+  ]
+};
+```
+
+The generated XML will not include the xmlns:video namespace:
+
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset
+  xmlns="http://www.sitemaps.org/schemas/sitemap/0.9"
+  xmlns:news="http://www.google.com/schemas/sitemap-news/0.9"
+  xmlns:xhtml="http://www.w3.org/1999/xhtml"
+  xmlns:image="http://www.google.com/schemas/sitemap-image/1.1"
+>
+  <!-- ... -->
+</urlset>
+```
+
+Note: The xmlns:video="..." line is missing from the XML.
```

Comment:
```suggestion
```
I don't think we need to call this out like this.

But we *could* consider a diff in the code sample maybe? We could have a `-` line that shows `xmlns:video` ? If so, can we show what the line would look like (in full, no `...` but the actual line Astro creates) if you hadn't excluded it? That could be a good example because then people can see what is cluttering up their XML that they didn't need!